common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -419,4 +419,8 @@ public long mergedIndexCacheSize() {
   public int ioExceptionsThresholdDuringMerge() {
     return conf.getInt("spark.shuffle.server.ioExceptionsThresholdDuringMerge", 4);
   }
+
+  public int appAttemptId() {
+    return conf.getInt("spark.app.attempt.id", -1);
```

Comment:
I think this's now is only valid for YARN. Could you comment it?

Reply:
@Ngone51 Curious if you mean about the assumption of parsing attempt as an `int` ? Or simply about this being used only in yarn ?

Reply:
Oh..I mean the latter one.